packages/cms-lib/lib/uploadFolder.js

```diff
@@ -13,19 +13,31 @@ const {
   logApiUploadErrorInstance,
   isFatalError,
 } = require('../errorHandlers');
+const { triggerNotify } = require('./notify');
 
 const queue = new PQueue({
   concurrency: 10,
 });
 
+async function uploadFile(portalId, file, destPath, apiOptions, notify) {
+  const uploadPromise = upload(portalId, file, destPath, apiOptions);
+
+  triggerNotify(notify, 'Uploaded', file, uploadPromise);
+  uploadPromise.then(() => {
+    logger.log('Uploaded file "%s" to "%s"', file, destPath);
+  });
+
+  return uploadPromise;
+}
+
 /**
  *
  * @param {number} portalId
  * @param {string} src
  * @param {string} dest
  * @param {object} options
  */
-async function uploadFolder(portalId, src, dest, { mode, cwd }) {
+async function uploadFolder(portalId, src, dest, { mode, cwd, notify }) {
```

Comment:
What happens if `notify` is not passed?

Reply:
There is no file to notify, so the watch works normally without notifying anything

Reply:
That makes sense. I misread how it was working. I do think that instead of adding `notify` support to `uploadFolder` we should either leverage the `uploadFolder` promise that is returned and do something like `uploadFolder.then(notify)` or not notify after the initial upload. Given that the use case is to live reload browser window(s) after changes are saved and uploaded while developing, I don't think there is much value in notifying after the initial upload, so I think we should remove it.

Reply:
I thought about this as we would miss the case where someone is uploading new files for the first time. Should we ignore that case in favor of not having a refresh in cases where the files are not new?

Reply:
> Should we ignore that case in favor of not having a refresh in cases where the files are not new?
Yeah, I think that is fine.

Reply:
After testing out the flow a bit with browser-sync, I tend to agree with your comment about not notifying after the initial upload, so I removed it.